### DISCUSSION TITLE
Failed to retrieve organizations

### DISCUSSION THREAD

**Discussion by IsoDevMate** [supabase/supabase#16691] [Questions] [answered]
just woke up only to try log into my project to get my api keys  and found the error below                                                                           
               Failed to retrieve organizations
Try refreshing your browser, but if the issue persists, please reach out to us via support.
[Contact support]

**Comment by encima** [comment, marked as answer]
We have an ongoing incident, you can view these (and subscribe) on our [status page](https://status.supabase.com/)
